pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -57,6 +58,11 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	fmt.Printf("Latest Kubernetes version: %s\n", latestVersion)
 	fmt.Printf("Current Node version: %s\n", nodeVersionInfoUpdate.Current.KubeletVersion.String())
 	fmt.Println()
+	target.IsSUSEOS()
+	if strings.Contains(target.Cache.OsRelease["VERSION_ID"], "15.1") {
+		fmt.Println("You are still running 15.1, and the upgrade to this branch expects running on SP2,\n please upgrade to the latest skuba for SP1, and migrate to SP2 before doing this.\n")
+		os.Exit(1)
```

Comment:
instead of os.Exit(1) and fmt.Println, you can simply log.Fatalf("")

Reply:
I did that. And the output was just uglier.